yarn.lock

```diff
@@ -2,115 +2,208 @@
 # yarn lockfile v1
 
 
+"@ampproject/remapping@^2.1.0":
```

Comment:
when you use a formal version of hof ensure to revert this file back to the original yarn lock and run `yarn add hof@latest` to avoid mutating the yarn lock. Just sticking this here in case you hadn't for this test as it looks like a lot has changed in it, but it may be from updating hof.

Reply:
Ahh very good point